tests/Settings/TestGlobalStack.py

```diff
@@ -0,0 +1,546 @@
+# Copyright (c) 2017 Ultimaker B.V.
```

Comment:
[already discussed, good for now]. It makes more sense to have tests for CuraContainerStack too, that prevents test duplication.